File: решения/java/src/Lab17/Product.java
package lab17;

import java.math.BigDecimal;

/**
 * @author akryukov
 *         20.07.2017
 */
public class Product {
    private String id;
    private String description;
    private String productGroupId;
    private String unitId;
    private BigDecimal weight;
    private BigDecimal cost;
    private int quantity;

    public Product(String id, String description, String productGroupId, String unitId, BigDecimal weight, BigDecimal cost, int quantity) {
        this.id = id;
        this.description = description;
        this.productGroupId = productGroupId;
        this.unitId = unitId;
        this.weight = weight;
        this.cost = cost;
        this.quantity = quantity;
    }

    public String getId() {
        return id;
    }

    public String getDescription() {
        return description;
    }

    public String getProductGroupId() {
        return productGroupId;
    }

    public String getUnitId() {
        return unitId;
    }

    public BigDecimal getWeight() {
        return weight;
    }

    public BigDecimal getCost() {
        return cost;
    }

    public int getQuantity() {
        return quantity;
    }
}

File: решения/cs/Classes/lab17/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Classes.lab17
{
    public class Product
    {
        public String Id { private set; get; }
        public String Description { private set; get; }
        public String ProductGroupId { private set; get; }
        public String UnitId { private set; get; }
        public Decimal Weight { private set; get; }
        public Decimal Cost { private set; get; }
        public int Quantity { private set; get; }

        public Product(String id, String description, String productGroupId, String unitId, Decimal weight, Decimal cost, int quantity)
        {
            Id = id;
            Description = description;
            ProductGroupId = productGroupId;
            UnitId = unitId;
            Weight = weight;
            Cost = cost;
            Quantity = quantity;
        }
    }
}
